cmd/gapit/common.go

```diff
@@ -70,6 +72,24 @@ func getGapis(ctx context.Context, gapisFlags GapisFlags, gapirFlags GapirFlags)
 		return nil, log.Err(ctx, err, "Failed to connect to the GAPIS server")
 	}
 
+	go func() {
```

Comment:
Nit: perhaps add a comment saying this is keeping gapis ticking over, preventing the idle time out from closing gapis.